test/apps/iodemo/ucx_wrapper.h

```diff
@@ -94,9 +94,8 @@ class UcxContext {
     // Called when there is a fatal failure on the connection
     virtual void dispatch_connection_error(UcxConnection* conn) = 0;
 
-    virtual bool add_connection(UcxConnection *conn);
-
-    virtual bool remove_connection(UcxConnection *conn);
+    // Called when new server connection is accepted
+    virtual void dispatch_connection_accepted(UcxConnection* conn);
```

Comment:
IMO better to avoid functions in base class which are relevant only for one inheritor, `add_conection` looks more generic

Reply:
add/remove connection was intended to be private helper to manage conn_id hash and not virtual

Reply:
ok